packages/vats/src/types.js

```diff
@@ -38,11 +38,17 @@
  * @property {(key: string) => void} reserve Mark a key as reserved; will
  * return a promise that is fulfilled when the key is updated (or rejected when
  * deleted).
+ * @property {( key: string, newValue: unknown, newAdmin?: unknown) =>
+ *   Promise<any> } default Update if not already updated.  Return
+ *   existing value, or newValue if not existing.
  * @property {(
- *   key: string, newValue: unknown, newAdmin?: unknown) => void
+ *   key: string, newValue: unknown, newAdmin?: unknown) => Promise<any>
+ * } set Update only if already initialized. Reject if not.
+ * @property {(
+ *   key: string, newValue: unknown, newAdmin?: unknown) => Promise<any>
```

Comment:
Why return `newValue`?

As above:
```suggestion
 * @property {<T>(
 *   key: string, newValue: T, newAdmin?: unknown) => Promise<T>
```

Reply:
Okay, I won't return newValue.

Reply:
Looks like you made `set` and `update` return void but didn't change the runtime. 

https://github.com/Agoric/agoric-sdk/commit/ac12696a10372d66cff048afd78d6750d9771011#diff-42f4291b619264addc7a7e12a8902cf3c60e5f797e97b5e4fa2a1e0c01d111ffR101

https://github.com/Agoric/agoric-sdk/commit/ac12696a10372d66cff048afd78d6750d9771011#diff-42f4291b619264addc7a7e12a8902cf3c60e5f797e97b5e4fa2a1e0c01d111ffR122

Reply:
Oops.  I'll fix that right away!